blog.html

```diff
@@ -0,0 +1,277 @@
+    <title>Blog</title>
+  </head>
+  <body>
+    <header class="header">
+      <a class="logo" href="index.html">Binabox</a>
+      <nav class="navigation">
+        <ul class="navigation-list">
+          <li class="navigation-item">
+            <a href="index.html" class="nav-link">Home</a>
+          </li>
+          <li>
+            <a href="blog.html" class="nav-link">Blog</a>
+          </li>
+          <li>
+            <a href="catalog.html" class="nav-link">Catalog</a>
+          </li>
+          <li>
+            <a href="contact-us.html" class="nav-link">Contact Us</a>
+          </li>
+        </ul>
+      </nav>
+      <div>
+        <input type="text" size="2" />
+        <a href="link" class="button-discord" target="_blank">Discord</a>
+        <button class="button-blue" type="button">Connect</button>
+      </div>
+    </header>
+    <main>
+      <section class="name-page">
+        <h1>Our blog</h1>
+        <ul class="blog-nav">
+          <li>
+            <a href="index.html" class="blog-nav-link">Home</a>
+          </li>
+          <li>
+            <a href="blog.html" class="blog-nav-link">Our blog</a>
+          </li>
+          <li>
+            <a href="basilisks.html" class="blog-nav-link">The Basilisks</a>
+          </li>
+        </ul>
+      </section>
+      <div class="blog-info">
+        <aside class="sidebar">
+          <div class="sidebar-box">
+            <p class="sidebar-title">Categories</p>
+            <ul class="sidebar-list">
+              <li><a href="#" class="sidebar-link">Our blog (12)</a></li>
+              <li><a href="#" class="sidebar-link">Metaverse (24)</a></li>
+              <li><a href="#" class="sidebar-link">Gaming (24)</a></li>
+              <li><a href="#" class="sidebar-link">Art (24)</a></li>
+              <li><a href="#" class="sidebar-link">Robot (24)</a></li>
+              <li><a href="#" class="sidebar-link">Lauchpad (24)</a></li>
+              <li><a href="#" class="sidebar-link">Crypto (430)</a></li>
+            </ul>
+          </div>
+          <div class="sidebar-box">
+            <p class="sidebar-title">Tag</p>
+            <ul class="sidebar-list">
+              <li><a href="#" class="sidebar-link">NFT</a></li>
+              <li><a href="#" class="sidebar-link">ICO</a></li>
+              <li><a href="#" class="sidebar-link">Robot</a></li>
+              <li><a href="#" class="sidebar-link">Crypto</a></li>
+              <li><a href="#" class="sidebar-link">Lauchpad</a></li>
+              <li><a href="#" class="sidebar-link">Metaverse</a></li>
+            </ul>
+          </div>
+        </aside>
+        <div class="blog-post">
+          <figure>
+            <img src="#" width="920" height="557" alt="dragon" />
+            <figcaption>Photo caption</figcaption>
+          </figure>
+          <h2>The Basilisks NFT</h2>
+          <button><a href="link" class="button-blue">Gaming</a></button>
```

Comment:
A link can't be inside a button and vice versa.
The 'button' tag would be better here, because an action should occur when clicked. Also we have to set the 'type' attribute for each button explicitly. If the button sends the form - then type="submit", otherwise type="button"

Reply:
done